.gitignore

```diff
@@ -42,3 +42,7 @@ htmlcov
 # Complexity
 output/*.html
 output/*/index.html
+
+# Local test venv
+venv/
+.venv/
```

Comment:
You should run tests with tox

Set up global gitignore for venv or whatever you use rather than try add it to every project you touch

Reply:
I do, but I also run tests in-IDE (VSCode) and default virtual envs created by tox are not auto-discovered by VSCode/Pycharm (they support `.venv`/`venv`). No worries, will remove it from here. I'm just used to see those in all the project I touch and didn't expect it to be a concern.

Reply:
I maintain 32 projects, I want all their config files to be the same unless there's a good reason not to.

Reply:
I can relate :)